src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -64,6 +64,7 @@ public HubConnectionContext(ConnectionContext connectionContext, TimeSpan keepAl
         // Currently used only for streaming methods
         internal ConcurrentDictionary<string, CancellationTokenSource> ActiveRequestCancellationSources { get; } = new ConcurrentDictionary<string, CancellationTokenSource>(StringComparer.Ordinal);
 
+        // REVIEW: Cancellable? The caller has no way to cancel without it (they can't CancelPendingFlush).
         public virtual ValueTask WriteAsync(HubMessage message)
```

Comment:
Was a decision made one way or the other on this method?

Reply:
Ah no, good point. Paging @davidfowl.

I lean towards adding a token because normally you'd be able to cancel the backpressure wait on 
a pipe using `CancelPendingFlush`, but the person who calls this method doesn't have access to the pipe. They could abort the connection I suppose.

Reply:
Following our plan of doing the minimal work for RC, I'm going to put these on the API so they're there in RC. We don't have to do anything with them until we've decided exactly how we want them to work.